uint32 session_id
uint32 microphone_id
uint32 cspeech_id
time ts
Utterance[] utterances